app/src/organizers/models.py

```diff
@@ -0,0 +1,32 @@
+import re
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from users.models import User
+from conferences.models import Zosia
+
+def validate_phone_number(value):
+    phone_reg = r'^(?<!\w)(\(?\+?\d{1,3}\)?[ -])?[ -]?\d{3}[ -]?\d{3}[ -]?\d{3}(?!\w)'
```

Comment:
+ I don't think you need `(?<!\w)` and `(?!\w)` - `^<regex>$` should suffice
+ This regex accepts malformed numbers like `+48) -123 456-789`